Gemfile.rails42

```diff
@@ -0,0 +1,12 @@
+source 'https://rubygems.org'
+gemspec
+
+gem 'jruby-openssl', :platforms => :jruby
+
+group :test, :remote_test do
+  # gateway-specific dependencies, keeping these gems out of the gemspec
+  gem 'braintree', '>= 2.0.0'
+  gem 'oauth', '0.4.7'
+end
```

Comment:
In that case we should add them as development dependencies in the gemspec. Development dependencies will not be installed when doing `gem install active_merchant`, so they will only be installed for active_merchant development.